src/Auth.php

```diff
@@ -25,6 +25,11 @@
  */
 class Auth
 {
+    /**
+     * The current version of CodeIgniter Shield
+     */
+    public const SHIELD_VERSION = 'v1.0.0-beta.5';
```

Comment:
@kenjis Are these items created automatically?

![Screenshot 2023-03-15 115940](https://user-images.githubusercontent.com/9530214/225251386-b3d59304-7793-4212-bae4-6638e6738922.png)

Reply:
No. these are tags that a human specify when creating releases.

It may be confusing, but the version number is like `1.0.0-beta.4` and the tag is `v1.0.0-beta.4`.
At least SemVer defines like that. See https://semver.org/#is-v123-a-semantic-version

Reply:
thank you!